frontend/ui/widget/keyvaluepage.lua

```diff
@@ -435,31 +439,50 @@ function KeyValuePage:init()
                     self:goToPage(page)
                 end
             end,
+            ok_text = "Go to page",
         },
+        call_hold_input_on_tap = true,
         bordersize = 0,
-        margin = Screen:scaleBySize(20),
```

Comment:
Hard to track down, but it's the removal of this that causes the sizing change issue.
Of course, it should be removed so there's the same little/no margin on the bottom as in Menu.
But somehow, this 2*Screen:scaleBySize(20) (or may be a bit less) should be accounted in the available_height computation to get the same results as before (which might have been a hack or just luck...).